lib/resources/r4/document_reference.yaml

```
@@ -290,3 +296,54 @@ fields:
         "end": "2020-01-01T01:00:00.000Z"
       }
     }
+
+- name: docStatus
+  required: 'Yes'
+  type: code
+  action:
+    - update
+  description: The status of the underlying document.
+  note: Each status are supported for all access types&#58; `final`, `amended`, and `entered-in-error`. If either status or docStatus are updated to "entered-in-error", the document will be treated as a document that was entered in error, regardless of the values of the other fields.
```

You can probably change this to something more like: 
```suggestion
  note: Allowed values for docStatus are&#58; `final`, `amended`, and `entered-in-error`. If either status or docStatus are updated to "entered-in-error", the document will be treated as a document that was entered in error, regardless of the values of the other fields.
```